azerothcore/azerothcore-wotlk
# fix(DB/Creature) flags_extra for brewfest barker bunny's

## CHANGES PROPOSED:
-  change flags_extra for brewfest barker bunny's so they are only visible to gm's


## ISSUES ADDRESSED:
- Closes none



## TESTS PERFORMED:
- apply sql and restart server
- checked that each brewfest barker bunny is only visible to gm's
 



## HOW TO TEST THE CHANGES:

- apply sql and restart server
- .go c 240003-240010
- .gm off (check that brewfest barker bunny is invisible)
- .gm on (check that brewfest barker bunny is visible)
## KNOWN ISSUES AND TODO LIST:




## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
